package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@next/next-common-files",
+  "version": "0.0.0",
+  "description": "Loaders for common file types.",
+  "main": "./index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "jest",
+    "prerelease": "npm t && npm run lint",
+    "release": "standard-version"
+  },
+  "keywords": [
+    "next",
+    "loader",
+    "plugin",
+    "svg",
+    "images",
+    "raster",
+    "audio",
+    "video"
+  ],
+  "author": "João Silva <joao.silva@moxy.studio>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:moxystudio/next-common-files.git"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^8.1.0",
+    "@commitlint/config-conventional": "^8.1.0",
+    "eslint": "^6.3.0",
+    "eslint-config-moxy": "^7.1.0",
+    "file-loader": "^4.2.0",
```

Comment:
I think we do, because the default behavior is for `url-loader` to fallback to file-loader. The idea here is that the developer must opt into the base64 translation (as is explained in the README), or leave the plugin as-is to use `file-loader`. 

So, in truth, we're not just using `url-loader`, we're defaulting to `file-loader` using the former as a proxy of sorts.

Reply:
Also, in conversation with @satazor it was discussed that `url-loader` likely comes with `file-loader` as a dependency. However, that is not the case, as can be verified in their [package.json](https://github.com/webpack-contrib/url-loader/blob/1b9dbd1cb406d221877237a6712b8bb4c9b194af/package.json#L39).

Reply:
Further discussion here: https://github.com/webpack-contrib/url-loader/pull/76

Anyway, I think we still need to move file loader as a regular dep and not a dev dep.